### DISCUSSION TITLE
About License

### DISCUSSION THREAD

**Discussion by wpsnl** [videovillage/community#208] [Q&A] [answered]
Hello,
I am a freelancer with many devices (controllable devices) that I own. Since the license can only be registered on a limited number of devices, I may need to transfer the license frequently.

Is there any penalty for transferring the license registration frequently? For example, is there a specified period for re-registration after a license transfer, or any other restrictions related to this process?

**Comment by gregcotten** [comment, marked as answer]
No penalty for doing that
